forest_height.R

```diff
@@ -0,0 +1,25 @@
+# Function to decide height of forest plots based on the number of studies
+# and the output format
+# 
+# param studies
+# A number (an int). The number of studies to be plotted.
+# param tab 
+# A string representing the way the plot height is measured
+# "pixel" for png files or plots rendered in the app 
+# "inch" for pdf files
+#
+# return 
+# A number 
+# Height in pixels for png plots
+# Height in inches for pdf plots
+#
+# author
+# NVB
+
+forest_height <- function(studies, measure){
+  if (measure == "pixel") {
```

Comment:
Since every location that calls this function has a hard-coded "measure" string, it'd make more sense to have 2 separate functions, or even better, 1 "private" function which does the calculation based on given values, and 2 "public" functions which do the calculation. eg:

calculate_forest_height(studies, subTwentyHeight, incrementalHeight)
calculate_forest_height_pixel(studies)
calculate_forest_height_inch(studies)

The last 2 functions simply call through to the general function, giving the values relevant to that measurement type.

Reply:
I've edited this to create two functions (one for pixel & one for pdf) but I'm not sure what you mean by them calling a separate function to do the calculations.